### PR TITLE
project done

### PR DESCRIPTION
**Pull request by ElShaer398**

### CODE REVIEW

**Comment by ElShaer398** [comment]
Hi
Good work so far but the project needs some improvements.

1. You should use linters to check for any errors or styling problems.
2. Please stick to the colors provided in the template.

Kindly check any other required changes in comments.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+
+
+  
+</header>
+  
+<body>
+
+
+          <h1>Handcrafted, home-made masterpieces</h1>
```
Comment (ElShaer398) [inline index.html:41]: Kindly adjust your tabs for clean code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+<!-- FORM -->  
+
+<div id="content">
+  
+  <div id="materials">
+
+    <img class="fire" src="https://cdn-icons-png.flaticon.com/512/477/477420.png">
+
+    <h3 class="materials_h3">
```
Comment (ElShaer398) [inline index.html:67]: Kindly put h3 in one line.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
+html, body {
```
Comment (ElShaer398) [inline style.css:1]: You can remove fixed height and width for responsive design.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+  position: fixed;
+  top: 0%;
+  
+}
+
+#nav-bar{
+
+background-color:yellow;
+  width: 100%;
+  height: 50px;;
```
Comment (ElShaer398) [inline style.css:19]: There is an extra semicolon please remove it.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+flex-direction:column;  
+justify-content: center;
+  
+}
+
+.content_p{
+  width: 30%;
+}
+
+.trombone{
```
Comment (ElShaer398) [inline style.css:77]: Please use tabs in the right way for clean code.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+  
+}
+
+
+#end{
+
+
+  width: 100%;
+  height: 10%;
+  background-color: #333333;
```
Comment (ElShaer398) [inline style.css:120]: Please use the right color for the text to appear clearly.

File: style.css (hunk 4 above)
Comment (binhussen) [reply style.css:19]: - Kindly, be polite when you give suggestions and raise the reasons 👌👍

File: style.css (hunk 5 above)
[the remark this answers is not in the document]
Comment (binhussen) [reply style.css:77]: - Kindly be reasonable when you give suggestions.👌👍

File: style.css (hunk 6 above)
Comment (binhussen) [reply style.css:120]: - Kindly, mention the reason and why they have to change.👌👍

File: index.html (hunk 2 above)
Comment (binhussen) [reply index.html:67]: - Kindly mention the reason why you are giving the suggestion 👌👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @ElShaer398 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
- Kindly, be polite and reasonable when you give sessions 👌👍

## Optional suggestions
- I think it will be great if you use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @ElShaer398, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Polite words are used ✅
- Reasons for the changes are made known ✅
## Optional suggestions
- I think it will be good if you make use of emojis and gifs, it will make the reviewer to appear friendly.😎😊
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**